easyInterface/Diffraction/DataClasses/DataObj/Experiment.py

```diff
@@ -353,6 +446,10 @@ def __init__(self, name: str, wavelength: Base, offset: Base, phase: ExperimentP
         self.setItemByPath(['offset', 'tooltip'], EXPERIMENT_DETAILS['offset']['tooltip'])
         self.setItemByPath(['offset', 'url'], EXPERIMENT_DETAILS['offset']['url'])
 
+        self.setItemByPath(['field', 'header'], EXPERIMENT_DETAILS['field']['header'])
```

Comment:
`field` should probably be renamed into `magnetic_field`, because electric field could also be a part of metadata.

Reply:
Modified